node/WsTokens.js

```diff
@@ -0,0 +1,19 @@
+module.exports = class WsTokens {
+  constructor(mysqlCon) {
+    this.con = mysqlCon;
+  }
+
+  isValid(token, userId) {
+    return new Promise((resolve, reject) => {
+      this.con.query('SELECT count(*) AS count FROM ws_tokens WHERE token = ? AND user_id = ?;',
```

Comment:
This doesn't check the token `used` column so `used` tokens can still connect to the services

Reply:
Oops. :) Done.